src/AutoRest.CSharp/Common/Utilities/StringExtensions.cs

```diff
@@ -322,7 +322,7 @@ public static string ToPlural(this string single, bool inputIsKnownToBeSingular
         public static string LastWordToPlural(this string single, bool inputIsKnownToBeSingular = true)
         {
             var words = single.SplitByCamelCase();
-            var lastWord = words.LastOrDefault();
+            var lastWord = words.Last();
```

Comment:
I think we want to leave LastOrDefault here, we never dereference `lastWord` and the comparison below should work with a null string fine.